pallets/stateful-storage/src/weights.rs

```diff
@@ -84,8 +84,10 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Proof: Msa DelegatorAndProviderToDelegation (max_values: None, max_size: Some(217), added: 2692, mode: MaxEncodedLen)
 	// Storage: unknown `0x0763c98381dc89abe38627fe2f98cb7af1577fbf1d628fdddb4ebfc6e8d95fb1` (r:1 w:1)
 	// Proof Skipped: unknown `0x0763c98381dc89abe38627fe2f98cb7af1577fbf1d628fdddb4ebfc6e8d95fb1` (r:1 w:1)
-	fn upsert_page(_s: u32, ) -> Weight {
-		Weight::from_ref_time(31_569_758 as u64)
+	fn upsert_page(s: u32, ) -> Weight {
+		Weight::from_ref_time(33_658_519 as u64)
+			// Standard Error: 306
+			.saturating_add(Weight::from_ref_time(1_379 as u64).saturating_mul(s as u64))
```

Comment:
Sometimes this happens since it's trying to find a linear regression. But since the page sizes are like 1024 it would be changeing the weight for about 1_500_000, so overall around the same weight